src/app/brukerprofil/adresse/MidlertidigAdresseUtland.tsx

```diff
@@ -0,0 +1,149 @@
+                    midlertidigAdresseUtland={this.props.midlertidigAdresseUtland}
+                />
+                <InputLinje>
+                    <div style={{flex: 4, marginRight: 15}}>
+                        <Input
+                            bredde={'XXL'}
+                            label="Adresselinje 1"
+                            defaultValue={this.props.midlertidigAdresseUtland.adresselinjer[0]}
+                            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                                onAdresselinjeChange(event, this.props, 0);
+                            }}
+                        />
+                    </div>
+                </InputLinje>
+                <InputLinje>
+                    <div style={{flex: 4, marginRight: 15}}>
+                        <Input
+                            bredde={'XXL'}
+                            label="Adresselinje 2"
+                            defaultValue={this.props.midlertidigAdresseUtland.adresselinjer[1]}
+                            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                                onAdresselinjeChange(event, this.props, 1);
+                            }}
+                        />
+                    </div>
+                </InputLinje>
+                <InputLinje>
+                    <div style={{flex: 4, marginRight: 15}}>
+                        <Input
+                            bredde={'XXL'}
+                            label="Adresselinje 3"
+                            defaultValue={this.props.midlertidigAdresseUtland.adresselinjer[2]}
+                            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                                onAdresselinjeChange(event, this.props, 2);
+                            }}
+                        />
+                    </div>
+                </InputLinje>
+                <InputLinje>
+                    <div style={{flex: 4, marginRight: 15}}>
+                        <Input
+                            bredde={'XXL'}
+                            label="Adresselinje4 "
+                            defaultValue={this.props.midlertidigAdresseUtland.adresselinjer[3]}
+                            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                                onAdresselinjeChange(event, this.props, 3);
+                            }}
+                        />
+                    </div>
+                </InputLinje>
+
+                <label className={'skjemaelement__label'}>Gyldig til</label>
+                <Datovelger
+                    dato={gyldigTil}
+                    id={'utlandsadresse-datovelger'}
+                    onChange={onGyldigTilChange(this.props)}
+                />
+            </>);
+    }
+}
+
+const mapStateToProps = (state: AppState): StateProps => {
+    return ({
+        landReducer: state.landReducer,
+        land: {input: '', feilmelding: ''}
```

Comment:
`mapStateToProps` er i utgangspunktet for på koble komponenten til Redux-staten vår, slik som for landReducer. For `land` så settes den alltid til blank og sendes til komponten som en `prop`. Dvs. at den alltid vil være det samme. Da tenker jeg at det er unødvendig å sette denne her 👍 Kanskje den heller kan komme via adressen i `OwnProps`?


![](https://media.giphy.com/media/Xk6awip87ICFG/giphy.gif)

Reply:
råger!